en/releases/1.14.md

```diff
@@ -0,0 +1,130 @@
+Other vehicles may need to configure output mappings when updating.
+:::
+
+The mixer file approach effectively hard-coded the geometry, and outputs that you had to use for each motor and actuator on a frame.
+Using [dynamic control allocation](../concept/control_allocation.md) is much more flexible because the configuration can depend on the airframe that you select.
+For example, selecting a generic airframe will allow you to specify both the geometry and output mapping, while selecting a particular ready-to-fly vehicle only allows you to test the output mappings (because everything else is hard coded/wired).
+Selecting a frame like an S-500 is somewhere in between: the geometry is fixed but you can map the flight controller outputs to whatever motors you like.
+
+:::note
+To use the new actuator configuration UI, you must have a recent release version of QGroundControl (version 4.2.0 or later).
+:::
+
+### Gazebo Simulator Changes
+
+Our [simulator](../simulation/README.md) naming convention has changed, because the [Open Source Robotics Foundation (OSRF)](https://www.openrobotics.org/) is in process of retiring the original Gazebo architecture:
+
+- The previous Gazebo architecture is now referred to as [Gazebo Classic](../sim_gazebo_classic/README.md#running-the-simulation), and the new architecture has taken the name [Gazebo](../sim_gazebo_gz/README.md) (the current release is "Gazebo Garden").
+- Gazebo Classic `make` targets are now prefixed with `gazebo-classic_` (e.g. `make px4_sitl gazebo-classic_cloudship`).
+- Gazebo targets are prefixed with `gz_` (e.g. `make px4_sitl gz_x500`).
+
+You can still use [Gazebo Classic](../sim_gazebo_classic/README.md#running-the-simulation) simulation in Ubuntu 20.04 or older.
+From Ubuntu 22.04 only Gazebo is supported and can be installed.
+
+Gazebo supports a number of vehicles and can also be used for multivehicle simulation.
+However the set of vehicles is still less diverse than available in Gazebo classic, so we'd very much welcome the contribution of more simulation targets.
+
+### ROS2 & XRCE-DDS Bridge
+
+[ROS2 support with micro XRCE-DDS bridge](../ros/ros2_comm.md) was added, replacing the [MicroRTPS bridge](https://docs.px4.io/v1.13/en/middleware/micrortps.html) middleware used in PX4v1.13 with [uXRCE-DDS](../middleware/uxrce_dds.md).
+
+:::note
+XRCE-DDS was an experimental opt-in feature in v1.13.
+:::
+
+The new architecture provides a more seemless integration with ROS 2, using the same message definition for both uORB and ROS2 topics, which are maintained in a single shared repository.
+It also provides a building block for microROS, which will allow ROS 2 nodes to be defined directly within PX4.
+
+ROS 2 users from PX4 v1.13 will need to use the new XRCE_DDS agent, and build their applications using the new message definitions.
+However they should generally not have to update their actual ROS 2 code.
```

Comment:
@beniaminopozzan Can you please check this section? I have rewritten it a bit from the original. What I am trying to capture is that when migrating you don't have to rewite your own code,.

I also want to capture the reasons we made the change. Essentially as precursor to microROS, but also to vastly simplify the dependencies - down to just the one shared message repo.

Reply:
Thanks @hamishwillee , you actually have to make small changes on your code to migrate to micro XRCE-DDS:

1. the topic names pattern changed. For example, before it was
   ```sh
   fmu/sensor_combined/out
   ```
   now it is
   ```sh
   fmu/out/sensor_combined
   ```
2. You have to adjust the QoS

Therefore the migration is not so effortless unfortunately.
And no, there is not a "migration guide" :(

Apart from that, it looks good to me!

Reply:
Thanks @beniaminopozzan . I integrated your specific examples. I think that is enough to show it won't be magic, but the changes should be constrained. A migration guide might be nice, but I think most things you need can be worked out from our existing docs

Reply:
@beniaminopozzan Actually, what if I just put those same comments in the ROS 2 Guide under a heading "Migrating from PX4 v1.13/MicroRTPS?" WHat else would we say?

FYI I'm going to merge this now. @junwoo091400 Still need remoteID info.

Reply:
@hamishwillee , I have to check and try myself upgrading an old ROS node written for MicroRTPS to the new version.
Give me a couple of days, I'll do that friday

Reply:
Thanks @beniaminopozzan - really appreciate how well you just sort stuff out. I am starting to wonder about your work-life balance!